builder/test-integration/spec/hypriotos-image/device-init_spec.rb

```diff
@@ -15,8 +15,8 @@
   its(:content) { should match /hostname: / }
   its(:content) { should match /docker:/ }
   its(:content) { should match /images:/ }
-  its(:content) { should match /- "\/var\/local\/rpi-consul_v0\.6\.4\.tar.gz"/ }
-  its(:content) { should match /- "\/var\/local\/rpi-swarm_v1.2\.1\.tar\.gz"/ }
+  its(:content) { should match /- "\/var\/local\/rpi-consul_v0.6.4.tar.gz"/ }
```

Comment:
good enough regex :-) But with this we can better grep for a specific version number in the code when updating to a newer release.